source/extensions/access_loggers/http_grpc/grpc_access_log_impl.h

```diff
@@ -24,63 +24,43 @@ namespace HttpGrpc {
 // TODO(mattklein123): Stats
 
 /**
- * Interface for an access log streamer. The streamer deals with threading and sends access logs
- * on the correct stream.
+ * Interface for an access logger. The logger provides abstraction on top of gRPC stream, deals with
+ * reconnects and performs batching.
  */
-class GrpcAccessLogStreamer {
+class GrpcAccessLogger {
 public:
-  virtual ~GrpcAccessLogStreamer() = default;
+  virtual ~GrpcAccessLogger() = default;
 
-  /**
```

Comment:
Please bring back interface doc comments. Also, what's your plan for eventually supporting TCP logs? Presumably just have a separate log message that takes a TCP log?

Reply:
The plan is to have a separate `log` method which accepts TCP log.
Most of the `log` function implementation will be moved to `flush` in next pull request, so http-specific logic is quite small.